src/test/java/tutoraid/model/lesson/LessonTest.java

```diff
@@ -28,7 +25,6 @@ public void isSameLesson() {
         Lesson editedMathsOne = new LessonBuilder(TypicalLessons.MATHS_ONE).withLessonName("Maths 1")
                 .withCapacity("51")
                 .withPrice("101")
-                .withStudents(new ArrayList<>(Arrays.asList()))
```

Comment:
Unfortunately the link is no longer tested (not so easy to make Student with Lessons / Lesson with Students to test